source/adapters/opencl/CMakeLists.txt

```diff
@@ -43,41 +43,24 @@ set_target_properties(${TARGET_NAME} PROPERTIES
 
 find_package(Threads REQUIRED)
 
-if (NOT DEFINED OpenCL_LIBRARY OR NOT DEFINED OpenCL_INCLUDE_DIR)
-    message(WARNING "OpenCL_LIBRARY and OpenCL_INCLUDE_DIR are not set. Using find_package() to find an OpenCL installation in the system.")
+# The OpenCL target can be set manually on upstream cmake to avoid using find_package().
+if (NOT OPENCL_TARGET)
```

Comment:
I think we should use the `UR_` prefix to avoid name collisions and be more explicit about the variable containing the path to the OpenCL-ICD-Loader library, something like: `UR_OPENCL_ICD_LOADER_LIBRARY`

Reply:
That makes sense. Updated the name in the latest commit